tinynn/graph/quantization/quantizable/gru.py

```diff
@@ -0,0 +1,341 @@
+from distutils.version import LooseVersion
+
+import numbers
+from typing import Optional, Tuple
+import warnings
+
+import torch
+from torch import Tensor
+
+if LooseVersion(torch.__version__) >= '1.13.0':
+
+    @classmethod
+    def from_float(cls, other, qconfig=None):
```

Comment:
It seems that the only difference between this one and the one in `lstm.py` is the actual layer type. What about refactoring the function a little bit?